logic/blacklistLogic.js

```diff
@@ -1,61 +1,57 @@
 const { BlacklistEntry } = require('../models');
 
 module.exports = class Blacklist {
-
-  static async augmentAllItems (allItems) {
+  static async augmentAllItems(allItems) {
     const blacklist = await BlacklistEntry.findAll({
       raw: true,
     });
-    return allItems.map(item => ({
+    return allItems.map((item) => ({
```

Comment:
why do we need the additional parentheses here?

Reply:
right now we have this rule in place https://eslint.org/docs/rules/arrow-parens 
tl;dr: its for consistency since we need parentheses in arrow functions with 2+ arguments

Reply:
I'd prefer single item without parentheses, what do you think?

Reply:
since the majority of our arrow functions only has a single argument I am happy to switch this